pkg/rtc/transport.go

```diff
@@ -284,6 +304,15 @@ func (t *PCTransport) createAndSendOffer(options *webrtc.OfferOptions) error {
 		return nil
 	}
 
+	if t.previousAnwser != nil {
+		t.previousAnwser = nil
+		if options == nil {
+			options = &webrtc.OfferOptions{ICERestart: true}
```

Comment:
```suggestion
			options = &webrtc.OfferOptions{}
```
Can we eliminate the else?

Reply:
Because peerconnection is negotiate with dumb pc first for sticky the data channel, so use ICERestart to connect with client

Reply:
Got it @cnderrauber . What I was suggesting was just a code style thing. Instead of
```
if options == nil {
    options = &webrtc.OfferOptions{ICERestart: true}
} else {
    options.ICERestart = true
}
```
can we do
```
if options == nil {
    options = &webrtc.OfferOptions{}
}
options.ICERestart = true
```

Reply:
Got it, Thanks!